frontend/GoodGrocer/app/screens/ActiveErrand.js

```diff
@@ -107,13 +105,13 @@ const ActiveErrand = ({ navigation }) => {
                 store: itemData.item.store,
               })
             }
+            key={itemData.item.id}
           />
         )}
         keyExtractor={(item) => item.id}
         ListHeaderComponent={() => (
           <View style={{ alignItems: "center" }}>
-            <Image source={require("../assets/logo.png")} />
-            <Text style={styles.title}>Current Errand</Text>
+            <Text style={styles.title}>Errand for {"COMMUNITY NAME HERE"}</Text>
```

Comment:
Should I pass in community info into the active errand response too?

Reply:
Yep that would be great :)